x/sequencer/keeper/unbond.go

```diff
@@ -65,6 +79,14 @@ func (k Keeper) forceUnbondSequencer(ctx sdk.Context, seqAddr string) error {
 
 	if oldStatus == types.Unbonding {
 		k.removeUnbondingSequencer(ctx, seq)
+	} else {
+		// in case the sequencer is currently reducing its bond, then we need to remove it from the decreasing bond queue
```

Comment:
same comment as before. maybe hook into `AfterSequencerUnbonded` (not jailed, as we can say that if he jailed he for sure is unbonded if he was bonded.)